docs/user_docs/intro/kusion-vs-x.md

```diff
@@ -6,20 +6,34 @@ sidebar_position: 2
 
-Terraform is a programmable operation product widely used in cloud resource delivery scenarios. It helps operators access different cloud APIs easily with programmable resource blocks, state management, and API providers. Terraform reduces the difficulty of using imperative APIs of various cloud vendors and provides a good declarative operation experience with a concise workflow.
+Kusion adopts your GitOps process and improves it with richness of features. The declarative [AppConfiguration](/docs/user_docs/concepts/appconfiguration) model can be used to express desired intent, once intent is declared [Kusion CLI](/docs/user_docs/reference/cli/kusion) takes the role to make production match intent as safely as possible. 
 
-In contrast, Kusion attempts to help developers work with an abstract app delivery model and also makes it easier for different roles to collaborate to build and manage the abstraction. Kusion natively supports scalability, automation, and high performance for enterprise-level requirements. Kusion works in a Kubernetes-first way and leverages Terraform to manage non-Kubernetes resources.
+**vs. PaaS (Heroku, Vercel, etc.)**
 
-**vs. CD systems (eg KubeVela, ArgoCD)**
+Kusion shares the same goal with traditional PaaS platforms to provide application delivery and management capabilities. The intuitive difference from the full functionality PaaS platforms is that Kusion is a client-side toolchain, not a complete PaaS platform. 
 
-CD systems are usually sourced in some declarative format and complete automated delivery and configuration drift reconciliation through Pull, Push, or the combination of Pull & Push. The Kusion engine can be regarded as a CD engine implementation in the push mode. If you have adopted a CD system, Kusion can be integrated with it, such as reconciling Kusion models through ArgoCD or writing codify OAM model and running with KubeVela, etc.
+Also traditional PaaS platforms typically constrain the type of applications they can run but there is no such constrain for Kusion which means Kusion provides greater flexibility.
+
+Kusion allows you to have platform-like features without the constraints of a traditional PaaS. However, Kusion is not attempting to replace any PaaS platforms, instead Kusion can be used to deploy to a platform such as Heroku.
+
+**vs. KubeVela**
+
+KubeVela is a modern software delivery and management control plane. KubeVela makes it easier to deploy and operate applications on top of Kubernetes.
+
+Kusion is not a control plane. Kusion is a tool for describing application intent in  a declarative way and providing consistent workflow to apply that desired state.
+
+With proper Generator implementation, the target Spec of [AppConfiguration](/docs/user_docs/concepts/appconfiguration) can be [KubeVela Application](https://kubevela.io/docs/getting-started/core-concept/) and Kusion can use KubeVela to satisfy the "apply" step.
```

Comment:
~satisfy~ execute the apply step

Reply:
actually Kusion will not call KubeVela directly to execute the apply step, KubeVela plays the role of Operator in this case,  so I am using `satisfy the "apply" step` here.